delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/model/ElasticSearchViewType.scala

```diff
@@ -36,4 +37,14 @@ object ElasticSearchViewType {
     override val toString: String = "AggregateElasticSearchView"
     override val tpe: Iri         = nxv + toString
   }
+
+  implicit final val esViewTypeEncoder: Encoder[ElasticSearchViewType] = Encoder.instance {
+    case ElasticSearch          => Json.fromString("ElasticSearch")
+    case AggregateElasticSearch => Json.fromString("AggregateElasticSearch")
+  }
+
+  implicit final val esViewTypeDecoder: Decoder[ElasticSearchViewType] = Decoder.decodeString.emap {
+    case "ElasticSearch"          => Right(ElasticSearch)
```

Comment:
these values do not match the Blazegraph ones...

Reply:
I've just changed it